docs/network_disruption/hosts.md

```diff
@@ -22,14 +22,19 @@ as NAT rules are applied before `tc` rules, and thus the port that a pod uses to
 It is not simple to detect that a hostname passed to `network.hosts` is a kubernetes service, and thus we include the `network.services` field.
 
 Whenever you want to disrupt traffic interacting with a kubernetes service[s], for correctness's sake, you _must_ specify the service under `network.services`, rather than `network.hosts`.
-`network.services` takes a list of services, which are defined with each service's `name` and `namespace`. The controller will take care of
-applying `tc` rules in a way that targets any port that may be used to talk to that service. There are no changes to how you should configure this field in a `node` level disruption
+`network.services` takes a list of services, which are defined with each service's `name` and `namespace`, as well as a list of `ports` to be affected. In this `ports` list, you can specify the `name` of the port, the `protocol` of the port and the `port` itself ([see kubernetes service definition for context](https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service)). 
+The controller will take care of applying `tc` rules in a way that targets any port that may be used to talk to that service. There are no changes to how you should configure this field in a `node` level disruption
 vs. a `pod` level disruption.
+
 ```
 network:
   services:
     - name: service_name
       namespace: example_namespace
+      ports:
+        - port: 8080 
+          name: service_port_name # optional. This is used to find the right port to be affected in case the service has multiple ports
+          protocol: tcp #optional. This is used to find the right port to be affected in case the service has multiple ports
```

Comment:
You may want to clarify what value will be used if this is not defined, I believe it's tcp